test/test_images/helloworld/release.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+image: ko://knative.dev/client/test/test_images/helloworld
```

Comment:
What does this file accomplish? It's not a valid K8s config so `ko apply` will fail. Does something else consume the output of `ko resolve` on this file?

If you just want to publish an image, you should be able to simply run `ko publish ./test/test_images/helloworld`

Reply:
The goal is build and publish the images to be used in tests. Currently there is only one image, we are planning to add more. So, if it is specified in this way , we could build and publish all images using 'ko resolve --jobs=4 ${tag_option} -RBf "${image_dir}" > /dev/null' .